authors/bruno-skvorc/polkadot/en_US/final.md

```diff
@@ -0,0 +1,268 @@
+
+A discussion on reducing shard count by a factor of 16 exists, which would drop the number to a mere 64 and approach the numbers that Polkadot is aiming for with parachains. This would improve composability (higher chance of contracts being on the same shard) and reduce the required number of validators for optimal security (128 * 64 is 8192, versus 131k before), but this is still under heavy [discussion](https://notes.ethereum.org/@vbuterin/HkiULaluS) and has other tradeoffs (notably - more bandwidth to run a validator, you'd need a stable 5Mbit+ connection).
+
+### Parachains
+
+Polkadot's Relay Chain is similar in concept to the beacon chain in Ethereum 2.0. It provides shared security to all the shards, here called **Parachains** (parallel chains). Parachains are independent blockchains which produce their own blocks and those blocks are then candidates for inclusion on the relay chain, much like crosslinks in Eth2. The main difference in parachains and shards is that parachains will be free to implement local logic any way they see fit. They will be application specific, but will share the security of both the relay chain and all other parachains.
+
+A chain optimized for DeFi use might exist, while another optimized for anonymous transactions might exist alongside it. Both will produce their blocks independently based on their own rules, but every X seconds where X is the target block time of the relay chain (depending on network performance - we're aiming for ~6 seconds) the hash of those blocks along with proofs that validators need to verify a legit state transition will be included into the relay chain.
+
+In Ethereum, each shard's _logic_ is identical to that of the other - more on that below. Parachains are added to Polkadot through an [auction system](https://wiki.polkadot.network/docs/en/learn-auction) - candidates bid on up to four 6-month intervals of parachain slot leases. DOTs of the lease auction's winner are locked up for the lease duration and returned afterwards. There will be few parachain slots to lease, so it should be a competitive market but this also means not every application-specific chain can expect to be able to get in on this scarce resource. Polkadot supports parachains as relay chains, so a nested relay chain architecture where a chain is another relay chain with its own parachains is also a possibility (similar to shards having shards in Ethereum 2). This part of Polkadot is still in the research phase.
+
+In other words:
+
+- Ethereum shards: identical (homogenous) chains with shared security, splitting the network's load by evenly distributing it
+- Parachains: different (heterogenous) chains with shared security, splitting the network's load by making sure each chain deals with its own specific context
+
```

Comment:
The **parachain section** needs some work in structure and also information about the availability and validity scheme. It should be mentioned that we are using much fewer miners than ETH2.0, and we can do it because of the availability and validity scheme. We should also mention that we assign validators randomly to parachains who are then responsible for its validity and availability. There should be a mention of fishermen there too. The auction scheme should e in a separate paragraph. There should be a note on how these design decisions impact security. Nested relay chains should ideally maybe not be mentioned at all since we still have to figure out how to do it, we had meetings about it but the design is not finalized. I would also avoid using "legit".